cypress/e2e/models/migration/applicationinventory/application.ts

```diff
@@ -396,9 +396,9 @@ export class Application {
             .within(() => {
                 click(selectBox);
                 cy.wait(SEC);
-                click('button[aria-label="Actions"]');
+                click('button[aria-label="Kebab toggle"]');
```

Comment:
Please move This selector to view .

Reply:
Done

Reply:
@ibragins , Just an FYI, this selector is already defined in applicationinventory.view.ts .